Example/src/AnimatedScrollExample.js

```diff
@@ -0,0 +1,279 @@
+  { artist: 'Radiohead', song: 'Creep' },
+  { artist: 'Simon & Garfunkel', song: 'Bridge Over Troubled Water' },
+  { artist: 'Aretha Franklin', song: 'Respect' },
+  { artist: 'Sly And The Family Stone', song: 'Family Affair' },
+  { artist: 'ABBA', song: 'Dancing Queen' },
+  { artist: 'The Beach Boys', song: 'Good Vibrations' },
+  { artist: 'Jimi Hendrix', song: 'Purple Haze' },
+  { artist: 'The Beatles', song: 'Yesterday' },
+  { artist: 'Chuck Berry', song: 'Jonny B Good' },
+  { artist: 'Bob Marley', song: 'No Woman No Cry' },
+  { artist: 'Jeff Buckley', song: 'Hallelujah' },
+  { artist: 'The Police', song: 'Every Breath You Take' },
+  { artist: 'The Beatles', song: 'A Day In The Life' },
+  { artist: 'Ben E King', song: 'Stand By Me' },
+  { artist: 'James Brown', song: "Papa's Got A Brand New Bag" },
+  { artist: 'The Rolling Stones', song: 'Gimme Shelter' },
+  { artist: 'Ray Charles', song: "What'd I Say" },
+  { artist: 'Dire Straits', song: 'Sultans Of Swing' },
+  { artist: 'The Beach Boys', song: 'God Only Knows' },
+  { artist: 'The Righteous Brothers', song: "You've Lost That Lovin' Feeling" },
+  { artist: 'The Who', song: 'My Generation' },
+  { artist: 'Martha Reeves and the Vandellas', song: 'Dancing In The Street' },
+  { artist: 'Prince', song: 'When Doves Cry' },
+  { artist: 'Sam Cooke', song: 'A Change Is Gonna Come' },
+  { artist: 'Ike and Tina Turner', song: 'River Deep Mountain High' },
+  { artist: 'The Emotions', song: 'Best Of My Love' },
+];
+
+data = data.slice(0, 10);
+
+const ITEM_SIZE = {
+  size: 250,
+  margin: 70,
+};
+const SCROLL_MARGIN = 20;
+const IPOD_MARGIN = 20;
+const SCREEN_WIDTH =
+  Dimensions.get('window').width - IPOD_MARGIN * 2 - SCROLL_MARGIN * 2;
+const BIG_BALL_SIZE = 200;
+const BIG_BALL_MARGIN = 0;
+const SMALL_BALL_SIZE = 50;
+const INNER_BALL_SIZE =
+  BIG_BALL_SIZE - SMALL_BALL_SIZE * 2 - BIG_BALL_MARGIN * 2;
+const DEFAULT_COVER_URI =
+  'https://e7.pngegg.com/pngimages/950/513/png-clipart-eighth-note-musical-note-stem-notes-music-download-graphic-arts.png';
+
+function ScrollExample() {
+  const position = useSharedValue(0);
+  const animatedRef = useAnimatedRef();
+
+  const itemTotalSize = ITEM_SIZE.size + ITEM_SIZE.margin * 2;
+  const borderMargin = SCREEN_WIDTH / 2 - itemTotalSize / 2 + ITEM_SIZE.margin;
+
+  const scrollToNearestItem = (offset) => {
+    'worklet';
+    let minDistance;
+    let minDistanceIndex = 0;
+    for (let i = 0; i < data.length; ++i) {
+      const distance = Math.abs(i * itemTotalSize - offset);
+      if (minDistance === undefined) {
+        minDistance = distance;
+      } else {
+        if (distance < minDistance) {
+          minDistance = distance;
+          minDistanceIndex = i;
+        }
+      }
+    }
+
+    scrollTo(animatedRef, minDistanceIndex * itemTotalSize, 0, true);
+  };
+
+  const scrollHandler = useAnimatedScrollHandler({
+    onScroll: (e, ctx) => {
+      position.value = e.contentOffset.x;
+    },
```

Comment:
Nitpick: we don't use `ctx` here so it should read `_ctx`

Reply:
actually, I think this is a matter of a coding convention, in some other files in our repo I spotted just `_` but as it is the last argument in that call we can equally just skip it there what's done here: 6578cd1